discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -750,6 +750,14 @@ def wait_for_cluster_to_be_in_finalizing_status(self):
             break_statuses=[consts.ClusterStatus.ERROR]
         )
 
+    def wait_for_cluster_to_be_in_status(self,statuses):
+        utils.wait_till_cluster_is_in_status(
+            client=self.api_client,
+            cluster_id=self.id,
+            statuses=statuses,
+            timeout=consts.CLUSTER_INSTALLATION_TIMEOUT,
```

Comment:
there are a few timeouts for multiple statuses, let it by passed as argument with default value

Reply:
/hold
unhold after fix

Reply:
/unhold

Reply:
/cc @lalon4